packages/mandb.rb

```diff
@@ -62,6 +60,7 @@ def self.install
   end
 
   def self.postinstall
-    system "env MANPATH=#{CREW_MAN_PREFIX} mandb -psc"
+    puts 'Creating mandb cache...'
+    system "unset MANPATH && mandb -C #{CREW_PREFIX}/etc/man_db.conf -psc --quiet"
```

Comment:
We're not going to deal with man page issues in upstream packages. It's not worth our time.

Actually... I'm going to rebuild this with more compress options from debian anyways...

Reply:
```
crew reinstall mandb
mandb: mandb is used to initialize or manually update index database caches that are usually maintained by man.
Performing pre-flight checks...
Precompiled binary available, downloading...
Mandb archive downloaded.
Unpacking archive using 'tar', this may take a while...
Removing since upgrade or reinstall...
Mandb removed!
Performing pre-install...
Performing install...
Using rdfind to convert duplicate files to hard links.
Now scanning ".", found 107 files.
Now have 107 files in total.
Removed 0 files due to nonunique device and inode.
Total size is 4079368 bytes or 4 MiB
Removed 107 files due to unique sizes from list. 0 files left.
Now eliminating candidates based on first bytes: removed 0 files from list. 0 files left.
Now eliminating candidates based on last bytes: removed 0 files from list. 0 files left.
Now eliminating candidates based on sha1 checksum: removed 0 files from list. 0 files left.
It seems like you have 0 files that are not unique
Totally, 0 B can be reduced.
Now making hard links.
Making 0 links.
Performing post-install for mandb...
Creating mandb cache. (Errors from this can either be ignored or reported upstream to the relevant package maintainers.)
Processing manual pages under /usr/local/share/man...
Updating index cache for path `/usr/local/share/man/man1'. Wait...mandb: iconv_open ("UTF-8//IGNORE", "ISO-8859-1"): Invalid argument
mandb: warning: /usr/local/share/man/man1/mmroff.1.gz: whatis parse for mmroff(1) failed
mandb: iconv_open ("UTF-8//IGNORE", "ISO-8859-1"): Invalid argument
mandb: warning: /usr/local/share/man/man1/grolbp.1.gz: whatis parse for grolbp(1) failed
Updating index cache for path `/usr/local/share/man/man3'. Wait...mandb: warning: /usr/local/share/man/man3/pcre2demo.3.zst: whatis parse for pcre2demo(3) failed
mandb: warning: /usr/local/share/man/man3/pcredemo.3.gz: whatis parse for pcredemo(3) failed
Updating index cache for path `/usr/local/share/man/man7'. Wait...mandb: iconv_open ("UTF-8//IGNORE", "ISO-8859-1"): Invalid argument
mandb: warning: /usr/local/share/man/man7/groff_mmse.7.gz: whatis parse for groff_mmse(7) failed
Updating index cache for path `/usr/local/share/man/man8'. Wait...mandb: warning: /usr/local/share/man/man8/saslauthd.8.gz: whatis parse for saslauthd(8) failed
done.
Processing manual pages under /usr/local/share/man/cat1...
Processing manual pages under /usr/local/share/man/cat5...
Processing manual pages under /usr/local/share/man/cat7...
Processing manual pages under /usr/local/share/man/cat8...
Processing manual pages under /usr/local/share/man/it...
Updating index cache for path `/usr/local/share/man/it/man8'. Wait...done.
11 man subdirectories contained newer manual pages.
8629 manual pages were added.
Mandb installed!
```